KerbalStuff/common.py

```diff
@@ -198,3 +199,13 @@ def check_mod_editable(mod, abort_response=401):
     if abort_response is not None:
         abort(abort_response)
     return False
+
+def get_version_size(f):
+    if not os.path.isfile(f): return None
+
+    size = os.path.getsize(f)
+    if size < 1023: return "%d %s" % (size, ( "byte" if size == 1 else "bytes" ))
+    elif size < 1048576: return "%3.2f KiB" % (size/1024)
+    elif < 1073741824: return "%3.2f MiB" % (size/1048576)
```

Comment:
Ouch! My suggestion had an error, apparently:
```suggestion
    elif size < 1073741824: return "%3.2f MiB" % (size/1048576)
```

Reply:
It was my mistake, I didn't pay attention to it.